source/installation/config.html.md

```diff
@@ -21,7 +21,8 @@ Here is a sample of what you should put inside for **GNU/Linux** and **Mac OS X*
 # (replace USER, PASSWORD, PORT and DATABASE_NAME with your values)
 #
 # If you are using SQlite, use the full path of the database file (`plume.db` for instance)
-#DATABASE_URL=\etc\path\to\Plume\plume.db
+# Windows user's paths are backslashes, change them to forward slashes
+#DATABASE_URL=/etc/path/to/Plume/plume.db
 DATABASE_URL=postgres://USER:PASSWORD@IP:PORT/DATABASE_NAME
```

Comment:
i would've assumed that the rust path library normalises the input

Reply:
I thought that as well, but if you do not change them, it will throw error when you go to start plume.